### ISSUE TITLE
Strip prefix when adding new users to groups

### ISSUE THREAD

**Issue by Wilfred** [anishathalye/hubot-group#13]
It would be nice if this worked:

```
wilfred> @hubot group add mygroup @bob
... currently no response from hubot ...
```

Currently, I have to write:

```
wilfred> @hubot group add mygroup bob
hubot> bob added to group mygroup
```

**Comment by anishathalye** [comment]
Good point. We'll fix this after #11 / #12 are sorted out.

**Comment by anishathalye** [comment]
Closing due to age, feel free to re-open if this is still an issue. My hubot deployments are currently in Keybase, not in Slack, so I don't know what the state of affairs is in Slack land. The HackMIT team is still using this bot in Slack, though, and I think it's working fine for them.
